### ISSUE TITLE
colour the mandatory columns

### ISSUE THREAD

**Issue by kmexter** [emo-bon/observatory-profile#27]
I think it would be helpful in the logsheets if all the mandatory columns had a colour. a visual clue to the stations to fill those in

**Comment by melanthia** [comment]
You are right. For all logsheets I make for other projects, I highlight, somehow, the mandatory fields by having the column title in BOLD, it also helps me.  Should we proceed with coloured columns in this case? I believe it will have no effect on your workflow either method applied.

**Comment by kmexter** [comment]
I think colour stands out better and indeed, it has no effect our our workflow

**Comment by kmexter** [comment]
maybe just colour the title row?

**Comment by melanthia** [comment]
Ok, I will give the logsheets some colour then :)

**Comment by melanthia** [comment]
so... I cannot apply any colour to the cells.. pfff...

**Comment by kmexter** [comment]
urm...can @cpavloud ? permission issue?

**Comment by melanthia** [comment]
I believe it's a permission issue, indeed. Let's see..

**Comment by kmexter** [comment]
indigo :-D

**Comment by isanti** [comment]
this is done; but I will re-check while I make other changes. Closing this as all changes are listed here as well: https://github.com/emo-bon/governance-data/issues/37
